website/_includes/donate-modal.html

```diff
@@ -2,39 +2,116 @@
+<div id="amount-modal" class="modal rounded max-w-2xl shadow-lg bg-white text-black-lighter fixed">
+  <div id="close-modal" class="absolute p-1 m-2">
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+      <path stroke="none" d="M0 0h24v24H0z" fill="none"/>
+      <line x1="18" y1="6" x2="6" y2="18" />
+      <line x1="6" y1="6" x2="18" y2="18" />
+    </svg>
+  </div>
+  <h1 class="text-blue-dark text-center font-4xl font-light">
+    {{ _('Pay What You Want') }}
+  </h1>
   <form action="">
-  {% if LANG in settings.LOCALE_CURRENCIES %}
-    {% set this_currency = settings.LOCALE_CURRENCIES[LANG] %}
-  {% else %}
-    {% set this_currency = 'usd' %}
-  {% endif %}
-  {% set this_curdata = settings.CURRENCIES[this_currency] %}
-    <p>Select an amount:</p>
-    <select name="currency-switcher-currency" id="id_currency-switcher-currency">
-    {% for currency, vals in settings.CURRENCIES | dictsort %}
-      <option value="{{ currency }}" {% if currency == this_currency %}selected{% endif %}>{{ currency.upper() }}   {{ vals.symbol.decode('utf-8') }}</option>
-    {% endfor %}
-    </select>
-    <input type="radio" id="amt1" name="amount" value="{{ this_curdata.presets[0] }}">
-    <label for="amt1">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.presets[0] }}</label><br>
-    <input type="radio" id="amt2" name="amount" value="{{ this_curdata.default }}" checked>
-    <label for="amt2">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.default }}</label><br>
-    <input type="radio" id="amt3" name="amount" value="{{ this_curdata.presets[2] }}">
-    <label for="amt3">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.presets[2] }}</label><br>
-    <input type="radio" id="amt4" name="amount" value="{{ this_curdata.presets[3] }}">
-    <label for="amt4">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.presets[3] }}</label><br><br>
-    <input type="radio" name="amount" value="">{{ _('Other') }}<input type="text" name="amount" />
-    <button id="amount-submit">{{ _('Donate & Download') }}</button>
-    <button id="amount-cancel">{{ _('No Thanks; Close & Download') }}</button>
+    <section class="description text-center">
+      <p>{{ _('Thunderbird is independent, free, and Open Source software sustained by your donations. Help us keep the project alive!') }}</p>
+    </section>
+    <section class="amount">
+      <div class="flex justify-between items-center">
+        <label class="uppercase font-sm">{{ _('Select an amount') }}</label>
+        <label>
+          {{ _('Currency') }}
+          <select name="currency-switcher-currency" id="id_currency-switcher-currency">
+            {% for currency, vals in settings.CURRENCIES | dictsort %}
+            <option value="{{ currency }}" {% if currency == this_currency %}selected{% endif %}>{{ currency.upper() }}   {{ vals.symbol }}</option>
```

Comment:
None of this actually runs due to the removal of the `.decode('utf-8')` part -- it was there for a reason :)

This might be a Python2 vs 3 issue if it was working for you, the server side site build is still using Python2. I've been meaning to correct this, can probably do it on Monday.

Reply:
Oh I'm sorry, I was indeed using Python 3.6 for development and forgot to put those decode statements back.

Both would be fine for me (developing in Python 2 or waiting for you to upgrade to Python 3) - let me know what works best for you. I won't work on this before Monday anyway.

Reply:
Yeah lets deal with this on Monday.

Reply:
Is this related to this error?
`UnicodeDecodeError: 'ascii' codec can't decode byte 0xc4 in position 1: ordinal not in range(128)`
I can't build it locally.

Reply:
@Alecaddd Yes it is. I made it Python2 compatible now - you should be able to build it again.